src/pages/createAppointment/[post_id].tsx

```diff
@@ -0,0 +1,254 @@
+        .then(() => {
+          router.push(PAGE_PATHS.MY_APPOINTMENTS);
+          return;
+        })
+        .catch((err: any) => {
+          console.log(err);
+          setIsCreatingAppointment(false);
+          return;
+        });
+    }
+  }
+
+  useEffect(() => {
+    if (!postId || !userStatus.user) return;
+    GetPostWithParticipantsByPostId(postId, supabaseClient)
+      .then((p: PostInfo) => {
+        setPostInfo(p);
+        if (!p.participants) return;
+        setAvailableParticipants(
+          p.participants.filter((e) => e.userId !== userStatus.user?.userId)
+        );
+      })
+      .catch((err: any) => {
+        console.log(err);
+        router.push(PAGE_PATHS.CREATE_APPOINTMENT);
+        return;
+      });
+  }, [supabaseClient, postId, userStatus.user, router]);
+
+  function backToPost(): void {
+    router.push(PAGE_PATHS.POST + router.query.post_id);
+    return;
+  }
+
+  if (userStatus.isLoading) return <Loading />;
+  if (!userStatus.user) {
+    router.push(PAGE_PATHS.LOGIN);
+    return;
+  }
+  if (userStatus.user.isAdmin) {
+    router.push(PAGE_PATHS.ADMIN_HOME + userStatus.user.userId);
+    return;
+  }
+  if (!userStatus.user.isVerified) {
+    router.push(PAGE_PATHS.HOME);
+    return;
+  }
+  if (!postInfo || !availableParticipants) return <Loading />;
+  return (
+    <>
+      <Navbar />
+      {isCreatingAppointment && <Loading />}
+      <IconButton
+        onClick={backToPost}
+        style={{ position: "absolute", top: 86, left: 20, zIndex: "1" }}
+      >
+        <ArrowBackIcon fontSize="large" color="secondary" />
+      </IconButton>
+
+      <Stack spacing={4} sx={{ marginBottom: "3vh" }} alignItems="center">
+        {/* Page header */}
+        <Box sx={{ marginTop: "3vh" }}>
+          <Typography variant="h1">Create Appointment</Typography>
+        </Box>
+
+        <Stack spacing={5} direction="row">
+          <Card sx={CreatePostStyle.Card}>
+            <Stack spacing={0} alignItems="center" justifyContent="center">
+              {/* Post title */}
+              <Box style={CreatePostStyle.TextField}>
+                <TitleTextField
+                  name="title"
+                  header="Title"
+                  placeholder="This is Post Title"
+                  value={postInfo?.title ? postInfo?.title : ""}
+                  handleValueChange={() => {}}
+                  isErr={false}
+                  errMsg=""
+                  disabled={true}
+                />
+              </Box>
+
+              {/* Location */}
+              <Box sx={CreatePostStyle.TextField}>
+                <LocationTextField
+                  header="Location"
+                  placeholder="Enter Location"
+                  initialValue={postInfo?.location ? postInfo?.location : ""}
+                  onChange={() => {}}
+                  isErr={false}
+                  errMsg=""
+                  disabled={true}
+                />
+              </Box>
```

Comment:
dropdown arrow as suggest ka (if you decide not to change it, you can resolve all comments related to this ka, I just point out where it appear)